### DISCUSSION TITLE
The AlwayOnDisplay Problem

### DISCUSSION THREAD

**Discussion by Onewpus** [tribalfs/GalaxyMaxHzPub#212] [Q&A] [answered]
After installing the app and using it, every time I turn off the screen I can only activate AOD 1 time and after that it doesn't work anymore.  Can anyone help me.
[https://drive.google.com/file/d/1HRPWZxvtk0t38tzDm0qejNWK7iQI_cIx/view?usp=drive_link](url)

**Comment by tribalfs** [comment, marked as answer]
You probably disabled AOD on your power saving mode settings.
